cypress/integration/ta/queue_create_inperson_or_online.js

```diff
@@ -0,0 +1,139 @@
+import {checkInTA, createAndLoginTA, createQueue, taOpenOnline} from "../../utils";
+
+describe('Can successfully create queues', () => {
+    beforeEach(() => {
+        // Set the state
+        createAndLoginTA();
+        createQueue({
+            courseId: "ta.course.id",
+        });
+    });
+
+    /*
+    it('Creates an online queue via modal', () => {
```

Comment:
What's the story behind this commented out test?

Reply:
on github's percy, the post request to create queue in seeds was not working, but ill uncomment it and see if it works today :(

Reply:
yo what do you know e2e passed on github (it was already passing on my computer but its uncommented now